extras/lbryinc/client.go

```diff
@@ -38,16 +45,16 @@ const (
 
 // NewClient returns a client instance for internal-apis. It requires authToken to be provided
 // for authentication.
-func NewClient(authToken string) Client {
-	return Client{
-		ServerAddress: defaultAPIHost,
+func NewClient(authToken string) Caller {
+	return &Client{
+		serverAddress: defaultAPIHost,
 		AuthToken:     authToken,
-		Logger:        log.StandardLogger(),
+		logger:        log.StandardLogger(),
```

Comment:
I specifically like this pattern. Does it allow for specially configured logging per type?